base/reduce.jl

```diff
@@ -573,10 +573,11 @@ function mapreduce_impl(f, op::Union{typeof(max), typeof(min)},
     simdstop  = start + chunk_len - 4
     while simdstop <= last - 3
         # short circuit in case of NaN
-        v1 == v1 || return v1
-        v2 == v2 || return v2
-        v3 == v3 || return v3
-        v4 == v4 || return v4
+        # === true is there to ignore missing
```

Comment:
"ignore" isn't the right term actually since we short-circuit. I can change this to just add "or missing" to the existing comment for NaN.

Reply:
Just say something like "to handle missing"

Reply:
Actually "short-circuit" is even more accurate I think.